app/javascript/controllers/details_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "@hotwired/stimulus";
```

Comment:
The name details_controller is too generic, what if there is another details sections somewhere else ? How will you name in that case ?

Reply:
Named changed to module_lesson_toggler_controller